design/ceph-config-updates.md

```diff
@@ -0,0 +1,127 @@
+# Updating Rook's Ceph configuration strategy (Jan 2019)
+**Targeted for v1.0-v1.1**
```

Comment:
I would really like to see this for 1.0.. happy to help out answering any questions on this end.

The main caveat with all of this is that the mon-based config doesn't exist until mimic.  If we need to support luminous, we'll still need a path that crams all the other random stuff in ceph.conf.  (This won't affect any of the other mon bootstrap changes, though!)

Reply:
I'm shooting for 1.0, but there is a lot going on here. My conservative estimate is that the mon rework and another couple daemons can done by 1.0 (target: early-mid March) but that the entirety likely won't be complete until 1.1 (target: early-mid June).